packages/tabs/src/TabTitle.tsx

```diff
@@ -143,6 +140,23 @@ const listTitle = css`
     outline: none;
   }
 
+  // We create a pseudo element that's the width of the bolded text
+  // This way there's no layout shift on hover when the text is bolded.
+  &:before {
+    content: attr(data-text);
+    content: attr(data-text) / '';
```

Comment:
What is this duplicate content attribute doing? Locally, it doesn't seem to be doing much other than overriding the first instance of this property.

Reply:
Ditto for Menu

Reply:
My understanding of the slash is that it's alt text, not a fallback. Since we're hiding this from screen readers, _I think_ it's safe here to remove this.

Reply:
Ah, yeah you're right